tools/zetasql_helper/zetasql_helper/local_service/local_service_grpc.h

```diff
@@ -0,0 +1,19 @@
+#ifndef ZETASQL_HELPER_LOCAL_SERVICE_LOCAL_SERVICE_GRPC_H_
+#define ZETASQL_HELPER_LOCAL_SERVICE_LOCAL_SERVICE_GRPC_H_
+
+#include "zetasql_helper/local_service/local_service.grpc.pb.h"
+#include "zetasql_helper/local_service/local_service.pb.h"
+
+namespace bigquery::utils::zetasql_helper::local_service {
+
+    // Implementation of ZetaSql Helper LocalService Grpc service.
+    class ZetaSqlHelperLocalServiceGrpcImpl : public ZetaSqlHelperLocalService::Service {
+    public:
+
+        grpc::Status Hello(grpc::ServerContext *context, const HelloRequest *request, HelloResponse *response) override;
```

Comment:
These are just dummy methods right?

Reply:
Right! They will be removed or changed to a health check in future.